caramel/scripts/tool.py

```diff
@@ -79,20 +79,29 @@ def error_out(message):
 
 
 def print_list():
-    requests = models.CSR.valid()
+    valid_requests = (
```

Comment:
I like the change, but as mentioned in chat, and now in writing, this moves the "deep" knowledge of the database from the models.py to scripts, while keeping the other (slower) methods in models.py

While this is not exaclty the same as  .valid() on the model, it is code that should live in the model, simply because it's deep affinity with how the database-model looks.